snooty.toml

```diff
@@ -20,3 +20,11 @@ full-version = "{+version+}.4"
 api = "https://mongodb.github.io/mongo-java-driver/{+version+}/apidocs"
 rs-docs = "https://www.reactive-streams.org/reactive-streams-1.0.4-javadoc/org/reactivestreams"
 driver-source-gh = "https://github.com/mongodb/mongo-java-driver"
+current-url = "https://www.mongodb.com/docs/languages/java/reactive-streams-driver/current/"
+
+[[banners]]
+targets = ["*"]
+variant = "info"
+value = """\
+    This version of the documentation is archived and no longer supported. View the `current documentation <{+current-url+}>`__ to learn how to `upgrade your version of the {+driver-short+} <{+current-url+}/upgrade>`__.\
```

Comment:
There's a double slash causing the URL to 404 (trailing slash in the constant and a slash in the link)

```suggestion
    This version of the documentation is archived and no longer supported. View the `current documentation <{+current-url+}>`__ to learn how to `upgrade your version of the {+driver-short+} <{+current-url+}upgrade>`__.\
```